wherehows-frontend/app/utils/Search.java

```
@@ -110,6 +79,7 @@ public static ObjectNode generateElasticSearchCompletionSuggesterQuery(String fi
       Logger.error("suggest Exception = " + e.getMessage());
```

Probably want to re-throw the exception